x/authz/client/cli/tx.go

```diff
@@ -123,6 +123,19 @@ Examples:
 						return err
 					}
 
+					queryClient := staking.NewQueryClient(clientCtx)
+
+					res, err := queryClient.Params(cmd.Context(), &staking.QueryParamsRequest{})
+					if err != nil {
+						return err
+					}
+
+					for i := 0; i < len(spendLimit); i++ {
```

Comment:
There will be only one bond denom, so we should return error if `len(spendLimit) > 1`. And we can cross check against `0` index entry. This didn't strike in my last review, sorry for the double work